lib/v2/ycwb/maintainer.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/': ['TimWu007'],
```

Comment:
```suggestion
    '/:node': ['TimWu007'],
```
This should be the same as `lib/v2/ycwb/router.js`